### ISSUE TITLE
How to add codeblocks?

### ISSUE THREAD

**Issue by khia** [anstosa/tw5-markdown#4]
Am I doing something wrong?

I have following content on my wiki:
````

For example, a macro called italicise that takes a single parameter called text can be invoked in any of these ways:

```
<<italicise "Text to be made into italics">>
<<italicise text:"Text to be made into italics">>
<$macrocall $name="italicise" text="Text to be made into italics"/>
<$macrocall $name="italicise" text={{Title of tiddler containing text to be italicised}}/>
<$macrocall $name="italicise" text=<<textMaker "Another macro to generate the text to be italicised">>/>
```
````

However it renders into something very strange
````
&lt;&lt;italicise &quot;Text to be made into italics&quot;&gt;&gt;
&lt;&lt;italicise text:&quot;Text to be made into italics&quot;&gt;&gt;
&lt;$macrocall $name=&quot;italicise&quot; text=&quot;Text to be made into italics&quot;/&gt;
&lt;$macrocall $name=&quot;italicise&quot; text={{Title of tiddler containing text to be italicised}}/&gt;
&lt;$macrocall $name=&quot;italicise&quot; text=&lt;&lt;textMaker &quot;Another macro to generate the text to be italicised&quot;&gt;&gt;/&gt;
````

**Comment by anstosa** [comment]
Hmm interesting. Which version of tw5-markdown are you using? I'm not able to replicate that on 1.1.1

![screenshot_51](https://user-images.githubusercontent.com/568242/37104406-7cf9b6d6-21e1-11e8-98bc-efcaa4d7cc90.jpg)

**Comment by iilyak** [comment]
I downloaded empty tw5 from https://tiddlywiki.com/
Then I opened a page mentioned in README.md  http://demo.santosa.family/#tw5-markdown
I drag and dropped the $:/plugins/anstosa/tw5-markdown link to my wiki.
It got me version 1.1.3.

**Comment by iilyak** [comment]
Which is very strange because this repository version is 1.1.1

**Comment by iilyak** [comment]
<img width="643" alt="screen shot 2018-03-12 at 11 41 11 am" src="https://user-images.githubusercontent.com/9804420/37303004-5f603df4-25ea-11e8-962b-8a9ade9cca5c.png">

**Comment by iilyak** [comment]
Sorry for the confusion @khia and @iilyak is the same person.  :-)

**Comment by anstosa** [comment]
lol! I didn't even notice the account changed 😛 

Yeah it looks like what happened is I left this in a bad state. I was trying to refactor things so the plugin worked for serverside rendering and I broke some stuff in process. I updated the demo site but didn't update this repo or even my personal wiki. I rolled back that change and updated the demo site so everything should be in sync and 1.1.4 on the demo site should work as expected.

Let me know how it goes!
